log4cl.asd

```diff
@@ -79,9 +88,8 @@
                              (:file "test-speed")
                              (:file "test-file-category")
                              (:file "test-compat")
-                             (:file "test-regressions")))))
-
-(defmethod perform ((op test-op) (system (eql (find-system :log4cl/test))))
-  (let ((*package* (find-package :log4cl-test)))
-    (eval (read-from-string "(stefil:funcall-test-with-feedback-message 'log4cl-test::test)")))
-  (values))
+                             (:file "test-regressions"))))
+  :perform (test-op (operation coponent)
+             (let ((*package* (find-package :log4cl-test)))
+               (eval (read-from-string "(stefil:funcall-test-with-feedback-message 'log4cl-test::test)")))
```

Comment:
if we'll decide to move with the flow and abandon older ASDF versions then there is `uiop:symbol-call` (or simply symbol-call) to achieve the same result. Excerpt from the ASDF manual:
```
#-asdf3.1 (error "my-lib requires ASDF 3.1")
(defsystem "my-lib"
  :class :package-inferred-system
  :depends-on ("my-lib/interface/all"
               "my-lib/src/all"
               "my-lib/extras/all")
  :in-order-to ((test-op (load-op "my-lib/test/all")))
  :perform (test-op (o c) (symbol-call :my-lib/test/all :test-suite)))
```